src/reduxCycles.js

```diff
@@ -0,0 +1,64 @@
+import xs from 'xstream';
+import {adapt} from '@cycle/run/lib/adapt';
+
+let actionListener = null;
+let stateListener = null;
```

Comment:
@kloy with new Cycle Unifed there are no more adapters so we won't have that problem of having to pass it as an argument.

@goshakkk how about calling it `reduxCycles` then?

```js
  const { createCycleMiddleware, makeActionDriver, makeStateDriver }  = reduxCycles();
  const store = createStore(
    rootReducer,
    applyMiddleware(createCycleMiddleware())
  );

  run(main, {
    ACTION: makeActionDriver(),
    STATE: makeStateDriver()
  })
```